### PR TITLE
Localizable implemented.

### LINKED ISSUE #17: Implement Localizable

### PR DESCRIPTION
**Pull request by maximschlei**
> Fixed Localizable Strings
> Helper method for MainBundle else PodBundle implemented

closes #17

### CODE REVIEW

File: SCIBarcodeScanner/Resources/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"camera.authorization.title" = "Kamerazugriff";
+"camera.authorization.reason" = "Bitte erlauben Sie den Zugriff auf die Kamera in den Einstellungen, damit der Barcodescanner funktioniert.";
+"camera.authorization.settings" = "Einstellungen";
+"generic.cancel" = "Abbrechen";
+
+
```
Comment (jakobvogel) [inline SCIBarcodeScanner/Resources/de.lproj/Localizable.strings]: Remove excessive white space

File: SCIBarcodeScanner/Resources/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,6 @@
+"camera.authorization.title" = "Kamerazugriff";
+"camera.authorization.reason" = "Bitte erlauben Sie den Zugriff auf die Kamera in den Einstellungen, damit der Barcodescanner funktioniert.";
+"camera.authorization.settings" = "Einstellungen";
+"generic.cancel" = "Abbrechen";
```
Comment (jakobvogel) [inline SCIBarcodeScanner/Resources/de.lproj/Localizable.strings]: Should we really use these MIOesque keys? (See main comment in delegate.)

File: SCIBarcodeScanner/View/SCIBarcodeScannerView.swift [hunk 3]
```diff
@@ -8,13 +8,40 @@ public protocol SCIBarcodeScannerViewDelegate {
     func sciBarcodeScannerPermissionMissing()
 }
 
+enum AlertStrings: String {
+    case title = "camera.authorization.title"
```
Comment (jakobvogel) [inline SCIBarcodeScanner/View/SCIBarcodeScannerView.swift]: Should we really use these MIOesque keys?

File: SCIBarcodeScanner/Resources/en.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,5 @@
+"camera.authorization.title" = "Camera access";
+"camera.authorization.reason" = "In order for the barcode scanner to work, please allow access to the camera in the settings.";
+"camera.authorization.settings" = "Settings";
+"generic.cancel" = "Cancel";
```
Comment (jakobvogel) [inline SCIBarcodeScanner/Resources/en.lproj/Localizable.strings]: Should we really use these MIOesque keys? (See main comment in delegate.)

File: SCIBarcodeScanner/View/SCIBarcodeScannerView.swift [hunk 5]
```diff
@@ -90,36 +117,39 @@ public class SCIBarcodeScannerView: UIView {
...
 
+    private func createAlert() -> UIAlertController {
+        let alert = UIAlertController(title: Helper.getLocalizedStringFrom(key: self.alertTitle, backUpKey: AlertStrings.title.rawValue),
+                                      message: Helper.getLocalizedStringFrom(key: self.alertMessage, backUpKey: AlertStrings.message.rawValue),
+                                      preferredStyle: UIAlertController.Style.alert)
+        alert.addAction(UIAlertAction(title: Helper.getLocalizedStringFrom(key: self.alertConfirm, backUpKey: AlertStrings.confirm.rawValue), style: UIAlertAction.Style.default, handler: { (action) in
+            guard let settingsUrl = URL(string: UIApplication.openSettingsURLString) else { return }
+            if UIApplication.shared.canOpenURL(settingsUrl) {
+                UIApplication.shared.open(settingsUrl, options: [:], completionHandler: { (success) in
+                    print("Scanner opened settings")
```
Comment (jakobvogel) [inline SCIBarcodeScanner/View/SCIBarcodeScannerView.swift:141]: Should we also inform the delegate?

**Review by jakobvogel — COMMENTED** [review]
Generally a great solution! 👍

File: SCIBarcodeScanner/View/SCIBarcodeScannerView.swift (hunk 5 above)
Comment (jakobvogel) [reply SCIBarcodeScanner/View/SCIBarcodeScannerView.swift:141]: ah, just found #22

File: SCIBarcodeScanner/Resources/en.lproj/Localizable.strings (hunk 4 above)
Comment (maximschlei) [reply SCIBarcodeScanner/Resources/en.lproj/Localizable.strings]: I changed them to be general

File: SCIBarcodeScanner/View/SCIBarcodeScannerView.swift (hunk 3 above)
Comment (maximschlei) [reply SCIBarcodeScanner/View/SCIBarcodeScannerView.swift]: I changed them to be general

File: SCIBarcodeScanner/Resources/de.lproj/Localizable.strings (hunk 2 above)
Comment (maximschlei) [reply SCIBarcodeScanner/Resources/de.lproj/Localizable.strings]: I changed them to be general

File: SCIBarcodeScanner/Resources/de.lproj/Localizable.strings (hunk 1 above)
Comment (maximschlei) [reply SCIBarcodeScanner/Resources/de.lproj/Localizable.strings]: removed

File: SCIBarcodeScanner/Resources/de.lproj/Localizable.strings [hunk 6]
```diff
@@ -1,6 +1,4 @@
-"camera.authorization.title" = "Kamerazugriff";
-"camera.authorization.reason" = "Bitte erlauben Sie den Zugriff auf die Kamera in den Einstellungen, damit der Barcodescanner funktioniert.";
-"camera.authorization.settings" = "Einstellungen";
-"generic.cancel" = "Abbrechen";
-
-
+"ccamera.alert.title" = "Kamerazugriff";
```
Comment (jakobvogel) [inline SCIBarcodeScanner/Resources/de.lproj/Localizable.strings]: ccamera? ;) I'll fix it.

**Pull request merged**